src/AmbientSounds.Uwp/App.xaml.cs

```diff
@@ -440,117 +441,25 @@ private void SetAppRequestedTheme()
+    /// Builds the <see cref="IServiceProvider"/> instance with the required services.
     /// </summary>
+    /// <param name="appsettings">The <see cref="IAppSettings"/> instance to use, if available.</param>
+    /// <returns>The resulting service provider.</returns>
     private static IServiceProvider ConfigureServices(IAppSettings? appsettings = null)
     {
-        var client = new HttpClient();
-
-        var provider = new ServiceCollection()
-            .AddSingleton(client)
-            // if viewmodel, then always transient unless otherwise stated
-            .AddSingleton<SoundListViewModel>() // shared in main and compact pages
-            .AddTransient<ScreensaverViewModel>()
-            .AddSingleton<ScreensaverPageViewModel>()
-            .AddSingleton<SettingsViewModel>()
-            .AddSingleton<CataloguePageViewModel>()
-            .AddSingleton<FocusTaskModuleViewModel>()
-            .AddSingleton<PremiumControlViewModel>()
-            .AddSingleton<FocusTimerModuleViewModel>()
-            .AddSingleton<ShellPageViewModel>()
-            .AddSingleton<PlayerViewModel>() // shared in main and compact pages
-            .AddSingleton<SleepTimerViewModel>() // shared in main and compact pages
-            .AddSingleton<FocusHistoryModuleViewModel>()
-            .AddSingleton<VideosMenuViewModel>()
-            .AddSingleton<TimeBannerViewModel>()
-            .AddSingleton<UpdatesViewModel>()
-            .AddSingleton<InterruptionPageViewModel>()
-            .AddSingleton<InterruptionInsightsViewModel>()
-            .AddSingleton<DownloadMissingViewModel>()
-            .AddSingleton<ShareViewModel>()
-            .AddSingleton<MeditatePageViewModel>()
-            .AddSingleton<FocusPageViewModel>()
-            .AddSingleton<CompactPageViewModel>()
-            .AddTransient<ActiveTrackListViewModel>()
-            .AddSingleton<AppServiceController>()
-            .AddSingleton<PlaybackModeObserver>()
-            .AddSingleton<ProtocolLaunchController>()
-            // object tree is all transient
-            .AddTransient<IStoreNotificationRegistrar, PartnerCentreNotificationRegistrar>()
-            .AddTransient<IImagePicker, ImagePicker>()
-            .AddSingleton<IClipboard, WindowsClipboard>()
-            .AddSingleton<IAppStoreRatings, MicrosoftStoreRatings>()
-            // Must be transient because this is basically
-            // a timer factory.
-            .AddTransient<ITimerService, TimerService>()
-            // exposes events or object tree has singleton, so singleton.
-            .AddSingleton<IDispatcherQueue, WindowsDispatcherQueue>()
-            .AddSingleton<IFocusNotesService, FocusNotesService>()
-            .AddSingleton<IFocusService, FocusService>()
-            .AddSingleton<IFocusHistoryService, FocusHistoryService>()
-            .AddSingleton<IFocusTaskService, FocusTaskService>()
```

Comment:
Looks like this wasn't migrated. @Sergio0694 can you help me double check if anything else was missed?